newIDE/app/src/BehaviorsEditor/NewBehaviorDialog.js

```diff
@@ -120,81 +56,80 @@ export default function NewBehaviorDialog({
     installExtension
   );
 
-  const platform = project.getCurrentPlatform();
-  const behaviorsMetadata: Array<EnumeratedBehaviorMetadata> = React.useMemo(
+  const deprecatedBehaviorsInformation = getDeprecatedBehaviorsInformation();
+
+  const allInstalledBehaviorMetadataList: Array<SearchableBehaviorMetadata> = React.useMemo(
     () => {
-      return project && platform
-        ? enumerateBehaviorsMetadata(
-            platform,
-            project,
-            eventsFunctionsExtension
-          )
-        : [];
+      const platform = project.getCurrentPlatform();
+      const behaviorMetadataList =
+        project && platform
+          ? enumerateBehaviorsMetadata(
+              platform,
+              project,
+              eventsFunctionsExtension
+            )
+          : [];
+      return behaviorMetadataList.map(behavior => ({
+        type: behavior.type,
+        fullName: behavior.fullName,
+        description: behavior.description,
+        previewIconUrl: behavior.previewIconUrl,
+        objectType: behavior.objectType,
+        category: behavior.category,
+        // PlatformExtension don't have tags so the information from EventsFunctionsExtension is lost.
+        // TODO Add tags to PlatformExtension.
+        // TODO Handle tags in MetadataDeclarationHelper
+        // TODO Handle tags in enumerateBehaviorsMetadata
+        tags: [],
+      }));
     },
-    [project, platform, eventsFunctionsExtension, extensionInstallTime] // eslint-disable-line react-hooks/exhaustive-deps
+    [project, eventsFunctionsExtension, extensionInstallTime] // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Can you double check the eslint disabling is still useful/ok?

Reply:
Ah, because now the dialog is closed upon installation.